src/foam/nanos/auth/UpdateChildEmailSetting.java

```diff
@@ -0,0 +1,115 @@
+package foam.nanos.auth;
+
+import foam.core.FObject;
+import foam.core.X;
+import foam.dao.ArraySink;
+import foam.dao.DAO;
+import foam.dao.ProxyDAO;
+import foam.mlang.MLang;
+import foam.util.SafetyUtil;
+
+import java.util.ArrayList;
+import java.util.function.BiConsumer;
+
+public class UpdateChildEmailSetting extends ProxyDAO {
+
+  public UpdateChildEmailSetting(X x, DAO delegate) {
+    super(x, delegate);
+  }
+
+  @Override
+  public FObject put_(X x, FObject obj) {
+    Group group    = (Group) obj;
+    Group oldGroup = (Group) this.getDelegate().find_(x, group.getId());
+
+    if ( oldGroup == null ) {
+      return super.put_(x, obj);
+    }
+
+    if ( ! SafetyUtil.equals(group.getDisplayName(), oldGroup.getDisplayName()) ) {
```

Comment:
I would rather only update the child if the child value is set. 
This matches the behaviour of looking up the parent value when the child is not set.